docs/api/lights/DirectionalLightShadow.html

```diff
@@ -12,7 +12,7 @@
 
 		<h1>[name]</h1>
 
-		<div class="desc">TODO</div>
+		<div class="desc">Directional light with a shadow map, for casting shadows.</div>

```

Comment:
That's not correct. `DirectionalLightShadow` is a type of shadow (basically a camera a few more parameters).